hollow/src/main/java/com/netflix/hollow/api/producer/ProducerListenerSupport.java

```diff
@@ -68,81 +67,45 @@ static boolean isValidListener(HollowProducerEventListener l) {
         return LISTENERS.stream().anyMatch(c -> c.isInstance(l));
     }
 
-    private final CopyOnWriteArrayList<HollowProducerEventListener> eventListeners;
-
-    ListenerSupport() {
-        eventListeners = new CopyOnWriteArrayList<>();
+    ProducerListenerSupport() {
+        super();
```

Comment:
Super minor, but I think the no-argument `super` call is implicit.

Reply:
Dropped it